### PR TITLE
Fix clippy

### PR DESCRIPTION
**Pull request by nicholastmosher**
None of the changes belong to items that are publicly-visible outside of the crate, so we can do a simple patch version bump.

### CODE REVIEW

File: nj-core/src/class.rs [hunk 1]
```diff
@@ -137,7 +137,7 @@ pub trait JSClass: Sized {
             }
         })();
 
-        result.to_js(&js_env)
+        result.into_js(&js_env)
```
Comment (sehz) [inline nj-core/src/class.rs:140]: why naming changes? I thought this PR was about clippy changes

**Review by sehz — CHANGES_REQUESTED** [review]

File: nj-core/src/class.rs (hunk 1 above)
Comment (nicholastmosher) [reply nj-core/src/class.rs:140]: One of the clippy lints is making sure that methods have conventional names based on their `self` type. Methods that take `self` by ownership should have the form `into_X`. `to_X` is used for `&self`.

Comment (nicholastmosher) [reply nj-core/src/class.rs:140]: I also checked: this trait is not exposed to the outside of this crate, so this is not a breaking change.

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "node-bindgen"
-version = "4.2.2"
+version = "4.2.3"
```
Comment (sehz) [inline Cargo.toml:3]: There is no need to bump up this since only dependent have changed

**Review by sehz — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 2 above)
Comment (nicholastmosher) [reply Cargo.toml:3]: Got it, I'll change that back

File: Cargo.toml [hunk 3]
```diff
@@ -16,9 +16,9 @@ build = ["nj-build"]
 
 [dependencies]
 nj-sys = { path = "nj-sys", version = "3.0.0", optional = true }
-nj-core = { path = "nj-core", version = "4.1.0", optional = true  }
+nj-core = { path = "nj-core", version = "4.1.3", optional = true  }
```
Comment (sehz) [inline Cargo.toml:19]: Ditto for dependencies as well

**Review by sehz — CHANGES_REQUESTED** [review]

**Review by sehz — APPROVED** [review]

**Pull request merged**